Meta/Lagom/Tools/CodeGenerators/StateMachineGenerator/main.cpp

```diff
@@ -252,7 +252,7 @@ HashTable<String> actions(const StateMachine& machine)
     }
     if (machine.anywhere.has_value())
         do_state(machine.anywhere.value());
-    return move(table);
+    return table;
```

Comment:
This probably belongs into a different commit? I have no idea what it does, but it's probably fine.

Reply:
It prevents a -Wpessimizing-move warning when built with all the proper warnings. before this commit, the Code Generators were built with less warnings than the rest of the system. but fair, removing a move is a bit of a different change to a file move.